content/en/docs/reference/glossary/multicluster-mesh.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Multicluster Mesh
+---
+A [service mesh](#service-mesh) composed of services running within more than one underlying cluster.
+All clusters in a multicluster mesh are under the same administrative control and share the same service definitions.
```

Comment:
That's the definition of a service mesh, and should go there.

Reply:
We could have a glossary entry for "Multicluster" only which just means more than one cluster running Istio, and explain that it can be modeled as a single mesh or multiple meshes that are federated. But, if we also remove the entry for "service mesh", we have no concrete term to use when talking about the single mesh composed from multiple clusters topology. Nathan and I have been leaning in the opposite direction, i.e., define the term "Multicluster" as the single mesh case only. Mesh Federation is then defined as the multiple mesh case.

Maybe defining all three terms in the glossary would be best: Multicluster Mesh (single mesh), "Mesh Federation" (multiple mesh), and just "Multicluster" (defined as can be either a "Multicluster Mesh" or a "Mesh Federation".